test/infrastructure/container/docker.go

```diff
@@ -412,12 +408,6 @@ func (d *dockerRuntime) RunContainer(ctx context.Context, runConfig *RunContaine
 		return errors.Wrapf(err, "unable to get Docker engine info, failed to create container %q", runConfig.Name)
 	}
 
-	// mount /dev/mapper if docker storage driver if Btrfs or ZFS
-	// https://github.com/kubernetes-sigs/kind/pull/1464
-	if d.needsDevMapper(info) {
-		hostConfig.Binds = append(hostConfig.Binds, "/dev/mapper:/dev/mapper:ro")
-	}
-
```

Comment:
Let's just remove these lines. I believe that should be sufficient to fix this bug. It is on my Machine. Removing the rest of the functionality breaks Kubelet at least on my Fedora setup running `btrfs`

Reply:
Just to clarify, when you say "remove", you mean to remove them from _this diff_, correct? In other words, don't remove these lines at all, as this PR already does?